app/models/validators/in_progress_validator.rb

```diff
@@ -4,12 +4,14 @@ module Validators
   # Displays a warning if the requests on the plate have already been completed
   class InProgressValidator < ActiveModel::Validator
     def validate(presenter)
-      return true unless presenter.labware.any_complete_requests?
+      return true unless presenter.labware.any_non_create_asset_requests_completed?
 
+      first_request = presenter.labware.active_non_create_asset_requests.first
+
+      return true if first_request.nil?
       presenter.errors.add(
         :submission,
-        '(active) is not present for this labware. ' \
-          'Any further work conducted from this plate may run into issues at the end of the pipeline.'
+        "Requests of type #{first_request.request_type.name} have already been down the pipeline. "
```

Comment:
maybe
'have already been through the pipeline from this plate and were completed. No active submission requests found.' ?

Reply:
I think the message is intended to get across 2 things:
1. previous completed requests
2. no active submission

is that correct?

Reply:
a further thought, if that is correct, do we give a warning at all if only one of those things is true?